cluster/src/assembly/resources/conf/iotdb-cluster.properties

```diff
@@ -157,4 +157,11 @@ max_number_of_logs_per_fetch_on_disk=1000
 
 # When consistency level is set to mid, query will fail if the log lag exceeds max_read_log_lag
 # This default value is 1000
-max_read_log_lag=1000
\ No newline at end of file
+max_read_log_lag=1000
+
+# Max number of clients in a ClientPool of a member for one node.
+max_client_pernode_permember_number=1000
+
+# Wait for some timeout, if the old client is not released
+# A new client is created
```

Comment:
I think the annotation could be more clearer if it can be configurated by the user.
```suggestion
# If the number of connections created for a node exceeds  `max_client_pernode_permember_number`, you need to wait so much time for other connections to be released until timeout, or a new connection will be created.
```